### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by jalvarezit**
In an attempt to prevent issues that are not relevant (e.g https://github.com/yogeshojha/rengine/issues/893) updated the issue templates to use new [github issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms).

### CODE REVIEW

**Review by AnonymousWP — CHANGES_REQUESTED** [review]
I'd say there's more required, such as more environments (e.g. Docker Engine version, Docker Compose version, etc.). I'd recommend taking a look at the development stack and see what software is required.

**Comment by itasahobby** [comment]
Sure @AnonymousWP, added some more fields that may be of interest.

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        - **Docker**: 20.10.22
+        - **Docker compose**: 2.15.1
+        - **Browser**: Microsoft Edge 112.0.1722.58
+    value: |
+        - rengine:
+        - OS:
+        - python:
+        - docker:
+        - docker-compose:
+        - browser
```
Comment (AnonymousWP) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:53]:
```suggestion
        - reNgine: 
        - OS: 
        - Python: 
        - Docker Engine: 
        - Docker Compose: 
        - Browser: 
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+name: 🚀 Feature request
+description: File a bug/issue
+title: "<title>"
```
Comment (AnonymousWP) [inline .github/ISSUE_TEMPLATE/feature_request.yml:3]:
```suggestion
title: "feat: <title>"
```

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 3]
```diff
@@ -0,0 +1,65 @@
+name: 🐞 Bug
+description: File a bug/issue
+title: "[BUG] <title>"
```
Comment (AnonymousWP) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:3]:
```suggestion
title: "bug: <title>"
```

**Review by AnonymousWP — CHANGES_REQUESTED** [review]
Should be fine after these suggestions.

**Comment by itasahobby** [comment]
Alright

**Comment by itasahobby** [comment]
Any updates on this? @AnonymousWP

**Comment by AnonymousWP** [comment]
> Any updates on this? @AnonymousWP

Yogesh needs to merge this, but he'll likely do that after he released reNgine 2.0.

**Review by yogeshojha — APPROVED** [review]

**Comment by yogeshojha** [comment]
Excellent guys :heart:

**Pull request merged**